doc/debugging-queryservice-backpressure.md

```diff
@@ -0,0 +1,119 @@
+# Debugging Backpressure in the QueryService Updater
+
+This document is intended to give a general overview of the existing pipeline which feeds MediaWiki updates into the QueryService, as well as guidance on what to do when the queue is blocked or malfunctioning.
+
+## Lifecycle of an update
+
+When an update in MediaWiki is happening, the following steps will be taken until it's present in the QueryService:
+
+1. MediaWiki calls the Platform API's `/event/pageUpdate` endpoint, pushing information about the change. The Platform API persists these changes in the `event_page_updates` table.
```

Comment:
I think we could add some hint that this is special wbstack secret sauce and not some built int mediawiki thing